ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/JumpPass.scala

```diff
@@ -0,0 +1,22 @@
+package io.joern.ghidra2cpg.passes
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.EdgeTypes
+import io.shiftleft.passes.{CpgPass, DiffGraph}
+import io.shiftleft.semanticcpg.language._
+class JumpPass(cpg: Cpg) extends CpgPass(cpg) {
```

Comment:
We should use a `ParallelCpgPass` here that performs this action per method.

Reply:
Done